client/tests/e2e/specs/delayAfterFailure.js

```diff
@@ -0,0 +1,275 @@
+      .mhFirst()
+      .mhGetBody().then((mailBody) => {
+        const codedLink = mailBody.split('href=3D"')[1].split('">')[0]
+        const withoutEq = codedLink.replace(/=\r\n/g, '')
+        const validationLink = withoutEq.replace(/=3D/g, '=')
+        cy.visit(validationLink)
+      })
+    cy.get('h3')
+      .should('contain', 'Adresse courriel validée')
+    // Gets the confirmation email
+    cy.mhGetAllMails()
+      .mhFirst()
+      .mhGetRecipients()
+      .should('contain', Cypress.env('emailCandidat'))
+    cy.mhGetAllMails()
+      .mhFirst()
+      .mhGetSubject()
+      .should('contain', '=?UTF-8?Q?Inscription_Candilib_en_attente_de_v?= =?UTF-8?Q?=C3=A9rification?=')
+    // The admin validates the candidate via Aurige
+    cy.visit(Cypress.env('frontAdmin') + 'admin-login')
+    cy.get('[type=text]')
+      .type(Cypress.env('adminLogin'))
+    cy.get('[type=password]')
+      .type(Cypress.env('adminPass'))
+    cy.get('.submit-btn')
+      .click()
+    cy.get('.v-snack')
+      .should('contain', 'Vous êtes identifié')
+    cy.contains('import_export')
+      .click()
+    cy.get('.ag-overlay')
+      .should('contain', 'No Rows To Show')
+    const filePath2 = '../../../' + Cypress.env('filePath') + '/aurige.json'
+    const fileName2 = 'aurige.json'
+    cy.fixture(filePath2).then(fileContent => {
+      cy.get('.input-file-container [type=file]')
+        .upload({
+          fileContent: JSON.stringify(fileContent),
+          fileName: fileName2,
+          mimeType: 'application/json',
+        })
+    })
+    cy.get('.v-snack')
+      .should('contain', fileName2 + ' prêt à être synchronisé')
+    cy.get('.import-file-action [type=button]')
+      .click()
+    cy.get('.v-snack')
+      .should('contain', 'Le fichier ' + fileName2 + ' a été synchronisé.')
+    // Verifies that the candidate is validated
+    cy.get('.ag-cell')
+      .should('contain', Cypress.env('candidat'))
+    cy.get('.ag-cell')
+      .should('contain', 'Pour le 75, envoi d\'un magic link à ' + Cypress.env('emailCandidat'))
+    cy.mhGetMailsByRecipient(Cypress.env('emailCandidat'))
+      .mhFirst()
+      .mhGetSubject()
+      .should('contain', '=?UTF-8?Q?Validation_de_votre_inscription_=C3=A0_C?= =?UTF-8?Q?andilib?=')
+    // Disconnects from the app
+    cy.get('.t-disconnect')
+      .click()
+    cy.get('.v-snack')
+      .should('contain', 'Vous êtes déconnecté·e')
+    cy.url()
+      .should('eq', Cypress.env('frontAdmin') + 'admin-login')
+    // The candidate gets the link
+    cy.mhGetAllMails()
+      .mhFirst()
+      .mhGetRecipients()
+      .should('contain', Cypress.env('emailCandidat'))
+    cy.mhGetAllMails()
+      .mhFirst()
+      .mhGetSubject()
+      .should('contain', '=?UTF-8?Q?Validation_de_votre_inscription_=C3=A0_C?= =?UTF-8?Q?andilib?=')
+    cy.mhGetAllMails()
+      .mhFirst()
+      .mhGetBody().then((mailBody) => {
+        const codedLink = mailBody.split('href=3D"')[1].split('">')[0]
```

Comment:
Ça c'est pas terrible, on peut faire mieux, mais on verra plus tard : je veux bien que tu mettes un `// TODO: decode properly the href`